orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/handler/CompleteStageHandler.kt

```diff
@@ -179,6 +179,10 @@ class CompleteStageHandler(
         stage.context["cloudProvider"]?.let {
           id.withTag("cloudProvider", it.toString())
         } ?: id
+      }.let { id ->
+        stage.metricTags?.let {
+          id.withTags(stage.metricTags)
```

Comment:
Does this merge or replace the existing tags?

Reply:
it merge the tags provided as a parameter with existing ones (here is the [implementation of `withTags`](https://github.com/Netflix/spectator/blob/main/spectator-api/src/main/java/com/netflix/spectator/api/DefaultId.java#L71-L73) method)